### PR TITLE
SysV daemon

### PR DESCRIPTION
**Pull request by serges147**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+name: Test Workflow
+on: [push, pull_request]
+env:
+  LLVM_VERSION: 15
```
[the remark this answers is not in the document]
Comment (serges147) [reply .github/workflows/tests.yml:4]: I'll try

File: .vscode/settings.json
[the remark this answers is not in the document]
Comment (serges147) [reply .vscode/settings.json:1]: I use mix. For example, I like VSCode when it comes to editing .md files, but I rarely compile and run there preferring CLion instead.

File: init.d/ocvsmd
[the remark this answers is not in the document]
Comment (serges147) [reply init.d/ocvsmd:1]: I'm also ok with OCVSMD

[the remark this answers is not in the document]
Comment (lydia-at-amazon) [reply init.d/ocvsmd:1]: So many consonants lol. Could we break it up with a hyphen? `OC-VSMD`?

(Also, up to this point, we have always named OpenCyphal projects with a noun, e.g. Yakut, Yukon, Libcyphal. So, using an acronym is breaking with tradition in my opinion and I still like `Yeppoon`. That being said, I'm willing to commit to `OC-VSMD` if that is the consensus.)

**Pull request merged**